src/View/mainpage/MainPage.py

```diff
@@ -263,3 +280,53 @@ def format_data(self, size):
         size: The size of the DicomStackedWidget
         """
         self.dicom_axial_view.format_metadata(size)
+
+    def create_image_fusion_tab(self):
```

Comment:
Many lines in this function too long. Docstring also missing.

Reply:
Could you possibly suggest how many lines this function should be?

Reply:
Couple lines in this function still too long (301, 336, 338, 339)